packages/file/lib/src/backends/memory/memory_file_system.dart

```diff
@@ -197,9 +197,7 @@ class _MemoryFileSystem extends FileSystem
     return node.type;
   }
 
-  /// Gets the node backing for the current working directory. Note that this
-  /// can return null if the directory has been deleted or moved from under our
-  /// feet.
+  /// Gets the node backing for the current working directory.
```

Comment:
Since this doesn't return null, what does it do? Or is the situation not reachable any longer? 

(basically, if we now throw it would be good to mention that)

Reply:
I'm not really sure.  I was mostly making the documentation match the existing behavior, and I'm not sure how to reproduce the behavior from the comment.   I tried:

```dart
final MemoryFileSystem fs = MemoryFileSystem();
fs.directory('/foo').createSync();
fs.currentDirectory = '/foo';
fs.directory('/foo').deleteSync();
print(fs.directory('/foo').existsSync()); // Prints: false
print(fs.currentDirectory.path); // Prints: /foo
```
I alternatively can change this to:
```dart
DirectoryNode? get _current => findNode(cwd) as DirectoryNode?;
```
`_current` is used only by `_findNode` where it's assigned to a nullable variable, and later code already handles it being potentially `null`, so that might be safer.

Reply:
Changing it to `DirectoryNode?` seems safer, probably worth exploring.